pyglmnet/pyglmnet.py

```diff
@@ -232,7 +169,7 @@ def copy(self):
         """Return a copy of the object."""
         return deepcopy(self)
 
-    def qu(self, z):
+    def __qu(self, z):
```

Comment:
@jasmainak i followed these conventions: http://www.diveintopython.net/object_oriented_framework/private_functions.html

if you use a double underscore, you will note be able to call `object.__private_method()`